src/ts/containers/Settings/FeatureConfig.tsx

```diff
@@ -90,14 +90,15 @@ export const FeatureConfig = ({feature}: FeatureConfigProps) => {
     )
 }
 
-const FeatureConfigContainer = styled('div')`
-    padding-left: 30px;
-`
+const FeatureConfigContainer = styled('div')``
```

Comment:
should we just use div, or is it considered a good practice to use named components?

Reply:
just easier to find the tree; Plus easier to add styles if needed